app/uk/gov/hmrc/testuser/connectors/AuthLoginApiConnector.scala

```diff
@@ -207,6 +207,7 @@ object GovernmentGatewayLogin {
           }
         case SAFETY_AND_SECURITY        => organisation.eoriNumber map { eoriNumber => Enrolment("HMRC-SS-ORG", Seq(Identifier("EORINumber", eoriNumber))) }
         case IMPORT_CONTROL_SYSTEM      => organisation.eoriNumber map { eoriNumber => Enrolment("HMRC-ICS-ORG", Seq(Identifier("EoriTin", eoriNumber))) }
+        case MA                         => organisation.eoriNumber map { eoriNumber => Enrolment("HMRC-MA-ORG", Seq(Identifier("EORINumber", eoriNumber))) }
```

Comment:
Could this be a longer name than `MA`?

Reply:
It can, I may have to raise another PR as Oli forked this into his own space and I can't push there.  I'll pop a comment on the jira ticket once done.